rest/rest-resources/src/main/java/org/apache/brooklyn/rest/security/provider/LdapSecurityProvider.java

```diff
@@ -21,24 +21,32 @@
 import com.google.common.base.CharMatcher;
 import com.google.common.base.Function;
 import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Lists;
-import java.util.Arrays;
-import java.util.Hashtable;
-import java.util.List;
-import java.util.function.Supplier;
-import javax.naming.Context;
-import javax.naming.NamingException;
-import javax.naming.directory.InitialDirContext;
```

Comment:
Try to avoid moving import statements just because your IDE decided it was a good idea - it makes it hard to see what's been added

Reply:
it's so annoying, and IJ seems particularly bad about doing this.  there should be a mode "optimize / auto-add imports without rearranging" but i can't find it either.